react-charts/donut-utilization-chart/step3.md

```diff
@@ -1,42 +1,61 @@
-Now we'll create a simple donut utilization chart...
-
-## Task
-
-1) Navigate to the `src` folder and open `src/App.js`{{open}}
-
-2) Next, we'll set up the general structure of the chart and the necessary import statements.
-
-Copy the following code into the App.js file, replacing all of the content there:
-
-<pre class="file" data-filename="App.js" data-target="replace">
-import React from 'react';
-import "@patternfly/react-core/dist/styles/base.css";
-import './app.css';
-import {
-  ChartAxis,
-  ChartDonutThreshold,
-  ChartDonutUtilization,
-  ChartThemeColor
-} from '@patternfly/react-charts';
-
-function App() {
-  return (
-    &lt;div className=&quot;chart-container&quot;&gt;
-      &lt;ChartDonutUtilization
-        data={{ x: &#39;Storage capacity&#39;, y: 90 }}
-        title=&quot;90%&quot;
-        subTitle=&quot;of 100 GBps&quot;
-        height={230}
-        width={500}
-      /&gt;
-    &lt;/div&gt;
-  );
-}
-
-export default App;
+Add thresholds to the donut utilization chart.
+
+1) <strong>Make sure the `App.js` file is still open.</strong>
+
+2) <strong>Locate the `<Chart>` component.</strong>
+
+It will look like this:
+
+<pre class="file">
+&lt;ChartDonutUtilization
+  data={{ x: &#39;Storage capacity&#39;, y: 90 }}
+  title=&quot;90%&quot;
+  subTitle=&quot;of 100 GBps&quot;
+  height={230}
+  width={350}
+/&gt;
+</pre>
+
+3) <strong>Add a `thresholds` property to the `<Chart>` component.</strong>
+
+The dynamic portion of the chart will change colors when data points reach the given `thresholds` properties.  This will change the dynamic portion of the chart to red.
```

Comment:
```suggestion
The dynamic portion of the chart will change colors when data points reach the given `thresholds` property.  This will change the dynamic portion of the chart to red.
```